projects/ngneat/transloco-messageformat/README.md

```diff
@@ -41,15 +41,6 @@ import { TranslocoMessageFormatModule } from '@ngneat/transloco-messageformat';
 
 ```
 
-It then enables support for the following within your i18n translation files:
```

Comment:
why remove the usage example?

Reply:
It was duplicated. 
The usage example is still in the readme. 

If you check out the readme as it is now, you will see the usage example has been duplicated. I removed the second instance of it.